docs/source/index.rst

```diff
@@ -21,7 +21,7 @@ As an open source project, Model Bakery welcomes contributions of many forms. Ex
 Compatibility
 =============
 
-model_bakery supports Django >= 1.11 and Python >= 3.5
+model_bakery supports Django >= 1.11 and Python >= 3.6
 
```

Comment:
Could you tell why dropping support of 3.5? Django supports from 3.4 on.

Reply:
@anapaulagomes honestly, I was a bit lazy to set up a special Travis job that will cover the next case:
`py{35}-django{111,20,21,22}-{postgresql,sqlite}`
...and then to exclude all those from the other jobs.

This seems needed as [Django 3.0 dropped Python 3.5 support](https://docs.djangoproject.com/en/3.0/releases/3.0/#python-compatibility).

And I honestly don't think we'll introduce any time soon some Python 3.5 incompatible code (which is now in security-only releases and to be dropped completely in 2020).

Reply:
But we can figure it out, if you think is still good to have (not sure if many people/systems did not switch to 3.6).

Reply:
I agree with sticking to Django's current supported versions just to leave it clearer to the user. Besides that, we don't have that much of version specific code in our code base. So, I'm good with that.

Reply:
Actually, maybe we don't have to specify which Python versions the lib supports. Because if the docs say the lib supports Django >= 1.11, it's implicit that we support Python 2.7, for example. In this case, I think that implicit is better than explicit hahaha